prow.go

```diff
@@ -222,6 +236,9 @@ func (m *jobManager) newJob(job *Job) error {
 		if len(input.Version) == 0 {
 			continue
 		}
+		// OSD needs to resolve the pullspec itself from cluster version, so include it
+		// in the env vars.
+		prow.SetJobEnvVar(&pj.Spec, "CLUSTER_VERSION", input.Version)
```

Comment:
ugh, this is a net new var i wasn't aware of.  I would probably prefer it be named differently - is only OSD using it right now?

Reply:
Yes - It is only for OSD atm. Their API works based on published Cincinnati versions and can't accept an arbitrary sha. I kept the name generic because I thought future providers might have a similar limitation (e.g. cluster pools), but I'm certainly open to making it OSD specific. `OSD_CLUSTER_VERSION` ?

Reply:
OSD_INSTALL_CLUSTER_VERSION for now would be ideal, with a big TODO: "this needs to be reconciled with a longer term strategy".

dedicated needs to work with images eventually.  Can you verify with Jeremy that's just a point in time thing?

Reply:
The recommendation has been implemented in https://github.com/openshift/ci-chat-bot/pull/131 . References to OSD have been removed from this PR.